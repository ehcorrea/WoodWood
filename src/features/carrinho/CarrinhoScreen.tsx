import { View } from 'react-native';

export default function CarrinhoScreen() {
  return <View></View>;
}
