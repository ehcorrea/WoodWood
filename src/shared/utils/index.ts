export * from './carrinho/carrinho';
export * from './responsive/responsive';
