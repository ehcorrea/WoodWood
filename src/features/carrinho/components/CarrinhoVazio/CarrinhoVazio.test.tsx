import { render } from '@/test/utils';
import CarrinhoScreen from '../../CarrinhoScreen';

describe('<CarrinhoVazio/>', () => {
  test('quando renderizado', () => {
    const container = render(<CarrinhoScreen />);
    expect(container).toMatchSnapshot();
  });
});
