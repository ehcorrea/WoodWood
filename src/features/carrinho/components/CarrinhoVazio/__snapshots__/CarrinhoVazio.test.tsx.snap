// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CarrinhoVazio/> quando renderizado 1`] = `
<View
  style={
    [
      {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": "5%",
        "paddingLeft": "5%",
        "paddingRight": "5%",
        "paddingTop": "5%",
      },
      undefined,
    ]
  }
>
  <View
    style={
      [
        {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
        undefined,
      ]
    }
  >
    <Text
      size="huge"
      style={
        [
          {
            "color": "#1A1D26",
            "fontFamily": "SofiaPro-Bold",
            "fontSize": 43,
            "lineHeight": 71.5,
          },
          [
            {
              "alignSelf": "center",
              "position": "absolute",
              "textAlign": "center",
              "top": "10%",
              "zIndex": 1,
            },
            undefined,
          ],
        ]
      }
      weight="bold"
    >
      Atualmente seu carrinho est√° vazio...
    </Text>
    <Image
      resizeMode="center"
      source={
        {
          "testUri": "../../../src/assets/images/figure-empty-list.png",
        }
      }
      style={
        [
          {
            "height": "100%",
            "width": "100%",
          },
          undefined,
        ]
      }
    />
  </View>
</View>
`;
